### ISSUE TITLE
Binary tree crashes when a null value is inserted to the tree.

### ISSUE THREAD

**Issue by BrandonArmand** [BrandonArmand/Binari#13]
- [x] Bug
- [ ] Clean up
- [ ] Question
- [ ] Enhancement
- [ ] Suggestion
- [ ] Other
 
## What needs to be done.
Fix issue with null values breaking the canvas

## Why
It is a vital issue that breaks the page.

## How
When a null value is added to the tree in some way
```
let tree = new Node(50)
tree.insert(null)
tree.insert(15)
tree.insert(35)
tree.insert(75)
tree.insert(85)
tree.insert(65)
```

The visual on the right will crash, and the page will need to be refreshed. Find out the cause.

**Comment by sibasish14** [comment]
I think it will work if we just skip (basically, do nothing) the values that are not integers. We can add a checker at the beginning of the function. Can I fix it?

A separate issue: what you have implemented is a Binary Search Tree (a type of binary tree). Maybe you should update the name of the introduction from "Binary Tree" to "Binary Search Tree". Otherwise, change the code to "Binary Tree".
@BrandonArmand

**Comment by NKaty** [comment]
Hi. I found out the reason the tree crashes with a null value. It is p5.text() method in Canvas component's draw function:
```
p5.text(el.value, el.x, el.y + (6 + elementScale));
```
This method expects the first argument to be a string. In the code of the method in p5, there is a check for undefined, but no check for null.
```
if (typeof str === 'undefined') {
    return;
} else if (typeof str !== 'string') {
    str = str.toString();
}
```
So when toString() method is called, the crash happens. We can make a check for null and call p5.text() method only if value is not null:
```
if (el.value !== null) {
    p5.text(el.value, el.x, el.y + (6 + elementScale));
}
```
The behaviour of the tree will be similar as if we passed an empty string as an argument in tree.insert() method:
```
let tree = new Node(50)
tree.insert("")
tree.insert(15)
tree.insert(35)
```
If that's okay, I'll make a pull request.
